sources/advertisements/VRM New Core/Controllers/StartVRMGroupProcessingController.swift

```diff
@@ -0,0 +1,28 @@
+//  Copyright 2018, Oath Inc.
+//  Licensed under the terms of the MIT License. See LICENSE.md file in project root for terms.
+
+import Foundation
+import PlayerCore
+
+final class StartVRMGroupProcessingController {
+    
+    let dispatch: (PlayerCore.Action)->()
```

Comment:
`(PlayerCore.Action) -> ()` - looks like your space keyboard button is not working :(